metrics/tests/location_indicator/change_image/style.json

```diff
@@ -0,0 +1,104 @@
+{
+  "version": 8,
+  "metadata": {
+    "test": {
+      "width": 512,
+      "height": 256,
+      "operations": [
+        [
+          "addImage",
+          "puck_hat",
+          "puck_hat.png",
+          {"pixelRatio": 2.0}
+        ],
+        [
+          "addImage",
+          "puck",
+          "puck.png",
+          {"pixelRatio": 2.0}
+
+        ],
+        [
+          "addImage",
+          "puck_shadow",
+          "puck_shadow.png",
+          {"pixelRatio": 2.0}
+        ],
+        [ "sleep", 3000 ],
+        [
+          "addImage",
+          "puck_hat",
+          "puck_hat.png",
+          {"pixelRatio": 4.0}
+        ]
+      ]
+    }
+  },
+  "sources": {
+    "geojson": {
+      "type": "geojson",
+      "data": {
+        "type": "FeatureCollection",
+        "features": [
+          {
+            "type": "Feature",
+            "properties": {
+            },
+            "geometry": {
+              "type": "Point",
+              "coordinates": [
+              139.766707,
+                35.693055
+
+              ]
+            }
+          }
+        ]
+      }
+    }
+  },
+  "center": [ 139.766707, 35.693055 ],
+  "zoom": 16,
+  "pitch" : 0,
+  "bearing" : 42,
+  "layers": [
+    {
+      "id": "background",
+      "type": "background",
+      "paint": {
+        "background-color": "lightseagreen"
+      }
+    },
+    {
+      "id": "puck123",
```

Comment:
Do we need this layer and the corresponding geojson source for this test?

Reply:
Not necessarily, but it was useful in the previous debugging cycle to assess whether something was rendered or nothing at all. Since background is sometimes simply a glClear, adding a proper layer helped verifying that something was actually rendered.
But it can be removed.